deep_speech_2/README.md

```diff
@@ -1,146 +1,393 @@
+- [Inference and Evaluation](#inference-and-evaluation)
+- [Distributed Cloud Training](#distributed-cloud-training)
+- [Hyper-parameters Tuning](#hyper-parameters-tuning)
+- [Training for Mandarin Language](#training-for-mandarin-language)
+- [Trying Live Demo with Your Own Voice](#trying-live-demo-with-your-own-voice)
+- [Experiments and Benchmarks](#experiments-and-benchmarks)
+- [Released Models](#released-models)
+- [Questions and Help](#questions-and-help)
+
+## Prerequisites
+- Only support Python 2.7
+- PaddlePaddle the latest version (please refer to the [Installation Guide](https://github.com/PaddlePaddle/Paddle#installation))
 
 ## Installation
 
+Please install the [prerequisites](#prerequisites) above before moving on.
+
 ```
+git clone https://github.com/PaddlePaddle/models.git
+cd models/deep_speech_2
 sh setup.sh
 ```
 
-Please replace `$PADDLE_INSTALL_DIR` with your own paddle installation directory.
+## Getting Started
 
-## Usage
+Several shell scripts provided in `./examples` will help us to quickly give it a try, for most major modules, including data preparation, model training, case inference and model evaluation, with a few public dataset (e.g. [LibriSpeech](http://www.openslr.org/12/), [Aishell](https://github.com/kaldi-asr/kaldi/tree/master/egs/aishell)). Reading these examples will also help us understand how to make it work with our own data.
 
-### Preparing Data
+Some of the scripts in `./examples` are configured with 8 GPUs. If you don't have 8 GPUs available, please modify `CUDA_VISIBLE_DEVICE` and `--trainer_count`. If you don't have any GPU available, please set `--use_gpu` to False to use CPUs instead.
 
-```
-cd datasets
-sh run_all.sh
-cd ..
-```
+Let's take a tiny sampled subset of [LibriSpeech dataset](http://www.openslr.org/12/) for instance.
+
+- Go to directory
+
+    ```
+    cd examples/tiny
+    ```
+
+    Notice that this is only a toy example with a tiny sampled subset of LibriSpeech. If we would like to try with the complete dataset (would take several days for training), please go to `examples/librispeech` instead.
+- Prepare the data
+
+    ```
+    sh run_data.sh
+    ```
+
+    `run_data.sh` will download dataset, generate manifests, collect normalizer' statistics and build vocabulary. Once the data preparation is done, we will find the data (only part of LibriSpeech) downloaded in `~/.cache/paddle/dataset/speech/libri` and the corresponding manifest files generated in `./data/tiny` as well as a mean stddev file and a vocabulary file. It has to be run for the very first time we run this dataset and is reusable for all further experiments.
+- Train your own ASR model
+
+    ```
+    sh run_train.sh
+    ```
+
+    `run_train.sh` will start a training job, with training logs printed to stdout and model checkpoint of every pass/epoch saved to `./checkpoints/tiny`. We can resume the training from these checkpoints, or use them for inference, evaluation and deployment.
```

Comment:
```resume the training``` is not rigorous.

Reply:
"resume training" is ok, e.g. https://cn.mathworks.com/help/nnet/ug/resume-training-from-a-checkpoint-network.html